Bottom: 136d0e2486b01f0291972647989452f5160e1166
Top:    0cb20d5bb6a518fb7c80ca1087d75bfe462efe66
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 13:57:57 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 9be81f5..b695535 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -323,13 +323,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
